resources/views/missions/operations.blade.php

```diff
@@ -19,8 +19,8 @@
 
                 if (isAssigned) {
                     $.ajax({
-                        type: 'POST',
-                        url: '{{ url('/hub/missions/operations/remove-mission') }}',
+                        type: 'DELETE',
+                        url: "{{ url('/hub/operations/1/missions') }}",
                         data: {'id': caller.data('item') || -1},
```

Comment:
Should store operation ID rather than operation mission ID, allowing url to change to '/hub/operations/{operation}/missions/{mission}'